src/_tailwind.css

```diff
@@ -0,0 +1,14 @@
+@import 'tailwindcss';
+
+@theme {
+  --color-primary: #00393f;
+  --color-secondary: #005a5a;
+  --color-tertiary: var(--mat-sys-tertiary);
+  --color-quaternary: #007e8c;
+  --color-quinary: #cfe7cd;
+  --color-error: #ba1a1a;
```

Comment:
maybe var(--color-red-500) or var(--mat-sys-error) to avoid the hard-coded value?